s64da_benchmark_toolkit/prepare.py

```diff
@@ -176,18 +176,32 @@ def _load_pre_schema(self, conn):
                 conn.cursor.execute(pre_schema_file.read())
 
     def _load_license(self, conn):
-        license_path = '/s64da.license'
         try:
-            conn.cursor.execute(f'select swarm64da.load_license(\'{license_path}\')')
-            print(f'Loading S64 DA License {license_path}')
+            conn.cursor.execute(f'select swarm64da.show_license()')
+        except errors.InvalidSchemaName:
```

Comment:
I wonder a bit why this might happen, can think only of not being able to load the extension in which case it might have died already earlier?

Reply:
The extension is not loaded for eg. native_psql runs